### PR TITLE
doc: Fix html builds for distcheck

### PR DESCRIPTION
**Pull request by petervo**

### CODE REVIEW

File: doc/guide/Makefile-guide.am [hunk 1]
```diff
@@ -38,9 +38,12 @@ CLEANFILES += \
...
+doc/guide/html: $(GUIDE_DOCBOOK) $(GUIDE_INCLUDES) $(man_MANS) $(GUIDE_STATIC) $(GUIDE_XSLT)
 	$(AM_V_GEN) mkdir -p doc/guide/html/ && rm -rf doc/guide/html/* && \
-		cp $(srcdir)/doc/guide/static/* doc/guide/html/ && \
 		$(XMLTO) html -m $(srcdir)/doc/guide/gtk-doc.xsl -o doc/guide/html/ \
 			--searchpath $(abs_builddir):$(abs_srcdir):$(abs_builddir)/doc/guide \
-			$(srcdir)/$(GUIDE_DOCBOOK)
+			$(srcdir)/$(GUIDE_DOCBOOK) && \
+		cp $(srcdir)/doc/guide/static/* doc/guide/html/
+
+doc/guide/html/index.html:
```
Comment (stefwalter) [inline doc/guide/Makefile-guide.am]: Why is this necessary? Shouldn't we just update install-doc-local to be:

```
install-doc-local: doc/guide/html
```

and remove these lines?

**Comment by petervo** [comment]
Updated

File: Makefile.am [hunk 2]
```diff
@@ -186,7 +186,7 @@ include doc/Makefile-doc.am
 include doc/guide/Makefile-guide.am
 include doc/man/Makefile-man.am
 
-noinst_DATA += doc/guide/html/index.html
+noinst_DATA += doc/guide/html
 
 install-doc-local: doc/guide/html/index.html
```
Comment (stefwalter) [inline Makefile.am]: Doesn't this line need to change too?

Comment (petervo) [inline Makefile.am]: Without that change you get errors like:
No rule to make target 'doc/guide/html/index.html', needed by 'all-am'.  Stop.
